okhttp/src/main/java/okhttp3/OkHttpClient.java

```diff
@@ -316,27 +316,27 @@ private static SSLSocketFactory newSslSocketFactory(X509TrustManager trustManage
     }
   }
 
-  /** Default call timeout (in milliseconds). */
+  /** Default call timeout (in milliseconds). By default calls have no timeout. */
```

Comment:
This should probably clarify that individual timeouts may still apply

Reply:
_Default call timeout (in milliseconds). By default there is no timeout for complete calls, but there is for the connect, write, and read actions within a call._